cmd/bursa/new.go

```diff
@@ -16,8 +16,34 @@ package main
 
 import (
 	"github.com/blinklabs-io/bursa/internal/cli"
+	"github.com/spf13/cobra"
 )
 
-func cliMain() {
-	cli.Run()
+var (
+	output string
+)
+
+func newCommand() *cobra.Command {
+	newCommand := &cobra.Command{
+		Use: "new",
```

Comment:
`bursa new wallet` is describing a state vs an action. `bursa create wallet` is giving and action, but I think I prefer `bursa wallet create` where the immediate first sub-command is the subject rather than the action. It'll also more closely reflect the API path `/api/wallet/create`

Reply:
Just pushed a change that does that; Rename `new` command to wallet and add a create subcommand in that so that its `bursa wallet create` now.